optuna_dashboard/ts/components/CompareStudies.tsx

```diff
@@ -213,14 +213,30 @@ export const CompareStudies: FC<{
                         >
                           {`# ${study.study_id}`}
                         </Typography>
+                      </Box>
+                      <Box
+                        sx={{
+                          display: "flex",
+                          flexDirection: "row",
+                          width: "100%",
+                        }}
+                      >
```

Comment:
With the changes in this PR, the appearance is as follows. It seems a bit off to have only the `study_id` displayed on the second line. What do you think about moving the `study_id` in front of the 'study_name' on the first line?

<img width="429" alt="Screenshot 2023-08-07 19 09 48" src="https://github.com/optuna/optuna-dashboard/assets/5564044/8a96dab7-f935-4498-9449-d146b6b45c08">

Reply:
Fixed.